build/stf-run-ci/tasks/main.yml

```diff
@@ -206,7 +187,7 @@
         name: service-telemetry-operator
         namespace: "{{ namespace }}"
       spec:
-        channel: unstable
+        channel: "{{ stf_channel }}"
```

Comment:
Perhaps I missed it, but where is this set to "unstable" for a local build?

Reply:
It's the default https://github.com/infrawatch/service-telemetry-operator/blob/b46f215d5a45a4b0b3703c3c7e54d2a4316ba02f/build/stf-run-ci/defaults/main.yml#L64